journal/modernized/03-chapter-2.adoc

```diff
@@ -197,6 +204,8 @@ From your afflicted sister,
 [.signed-section-signature]
```

Comment:
I'm guessing now that @jaredh159  hadn't totally finished inventing all of the syntax in the style guide when he did Ann Branson. Because "From your afflicted sister," in line 202 should have a `[.signed-section-closing]` above it.